mimic/model/ironic_objects.py

```diff
@@ -219,15 +219,15 @@ def list_nodes(self, include_details):
         http://docs.openstack.org/developer/ironic/webapi/v1.html#get--v1-nodes-detail
         """
         if not self.ironic_node_store:
-            for each in range(3):
+            for each in range(30):
```

Comment:
style nit: if `each` is not used in the body of the loop, `_` can be used as a throwaway iterator variable. Helps express that the loop is run purely for the side effect.